src/decorators/index.ts

```diff
@@ -0,0 +1 @@
+export * from 'decorators/withOutdatedCheck';
```

Comment:
`decorators` folder name is a bit misleading, IMHO

Reply:
hmmm, maybe you're right. It's actually hight order function, but is it not the same? It's some "thing" that "decorates" function with additional logic in this case logging something before invoking the command.

Reply:
well, I would expect `withOutdatedCheck` to be a proper decorator, with `@` prefix and proper syntax, not a function that is actually a helper, in fact